packer/install-dependencies.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+CONTAINERD_DOWNLOAD_URL_BASE="https://storage.googleapis.com/cri-containerd-release/"
```

Comment:
The general re-purposing of `CONTAINERD_TGZ_TMP ` (which needs `CONTAINERD_DOWNLOAD_URL`, and thus `CONTAINERD_DOWNLOAD_URL_BASE`) in the CSE install script means that we have to pass in a value for `CONTAINERD_DOWNLOAD_URL_BASE` before we source the script.